src/include/utilites.hpp

```diff
@@ -60,6 +60,46 @@ bool isIntersectVector2VectorArray(const ShapeT& shape, const VectorShapeT& shap
 
 inline void updateGameScore(std::size_t* gameScore, const Food& food)
```

Comment:
Can you pass the game score by reference, not by pointer?

Reply:
It's recommended to pass arguments, which will be changed, by pointer.